src/main/resources/reference-data/uc-decision-notice-questions.txt

```diff
@@ -0,0 +1,700 @@
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_physicalDisabilities",
```

Comment:
should Fixed list Id be unique?

Reply:
Good spot - I've now prefixed with esa,  but as the lists are the same I think we should have a conversation about whether we want to share a list